aten/src/THC/THCBlas.cu

```diff
@@ -238,6 +238,31 @@ void adjustLdLevel3(char transa, char transb, int64_t m, int64_t n, int64_t k, i
 
 }
 
+// Check https://gist.github.com/umanwizard/2b2e2fc12485ef6dc1cdfb1421276dd9
```

Comment:
you should link an issue, not a fixed gist (the issue can link to the fixed gist).  That way we can be more confident that if we get updated information, someone looking here would be able to find it.